arviz/plots/ppplot.py

```diff
@@ -108,24 +97,31 @@ def plot_pp(
     if backend_kwargs is None:
         backend_kwargs = {}
 
-    if data_pairs is None:
-        data_pairs = {}
-
     datasets = [getattr(data, group) for group in groups]
 
     if var_names is None:
         var_names = list(datasets[0].data_vars)
-    var_names = _var_names(var_names, datasets[0])
-    vars = [var_names]
-    for i in range(1, len(datasets)):
-        var_name =[data_pairs.get(var, var) for var in var_names]
-        var_name = _var_names(var_name,  datasets[i])
+
+    if isinstance(var_names, str):
+        var_names = [var_names]
+
+    if isinstance(var_names, list) or isinstance(var_names, tuple):
+        var_names = [list(var_names) for _ in datasets]
```

Comment:
`var_names` should allow string, list of strings and list of lists but end up being a list of lists always:

```
if isinstance(var_names, str):
   var_names = [var_names]
# now it is a list
if isinstance(var_names[0], str):
    var_names = [var_names for _ in datasets]
# now it is a list of lists the same length of datasets with the var names that correspond to the dataset
```

Reply:
Okay, I understand